### ISSUE TITLE
Popup Color /  clearer transition between features

### ISSUE THREAD

**Issue by meyerlor** [3liz/lizmap-web-client#4254]
### Is your feature request related to a problem? Please describe
In a Lizmap 3.7 popup it is visually very hard for me to distinguish where one feature ends / the other begins. Is it possible to change it like it was in 3.6 via HTML/CSS coding? If so - where? 

### Describe the solution you'd like
Have a clear "gap" between features in the Popup

### Describe alternatives you've considered
Going back to 3.6

### Additional context
See screenshots for better visualisation:
Lizmap 3.7:
![grafik](https://github.com/3liz/lizmap-web-client/assets/15894401/62bb16ad-d026-4524-99d8-46b2a25f39a0)

Lizmap 3.6:
![grafik](https://github.com/3liz/lizmap-web-client/assets/15894401/bfd0da6b-0a9e-4ef5-9fca-770e204dc05a)

**Comment by nboisteault** [comment]
I can't reproduce this issue here: https://demo.lizmap.com/lizmap/index.php/view/map?repository=features&project=multi_atlas

**Comment by meyerlor** [comment]
Yes, when having the automatic PopUp setting activated in thr QGIS Plugin, it is better. With Drag&Drop or HTML Maptip activated, it is like in my screenshot

**Comment by Gustry** [comment]
Automatic popup (or similar with the QGIS HTML maptip) works well.

It doesn't work well with D&D popup (either from QGIS HTML Maptip or from Form popup).

**Comment by meyerlor** [comment]
Great!
